Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -10099,21 +10103,21 @@ def remove_old_incidents_ids(found_incidents_ids, current_time, look_back):
 
 def get_found_incident_ids(last_run, incidents, look_back, id_field):
     """
-    Gets the found incident ids
+    Gets the found incident ids from the last run object and adds the new fetched incident IDs.
 
     :type last_run: ``dict``
     :param last_run: The LastRun object
 
     :type incidents: ``list``
-    :param incidents: List of incidents
+    :param incidents: List of incidents to add
 
     :type look_back: ``int``
     :param look_back: The look back time in minutes
 
     :type id_field: ``str``
     :param id_field: The incident id field
 
-    :return: The new incidents ids
+    :return: The new incident ids object after removing old incident IDs
```

Comment:
```suggestion
    :return: The new incident ids.
```
this line only specifies what's returned from the method.